### PR TITLE
PBM-447 Add a function to field-test compression and upload speeds

### PR DESCRIPTION
**Pull request by dAdAbird**

### CODE REVIEW

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| mongodb 3.6 | passed |
| mongodb 4.0 | passed |
| mongodb 4.2 | passed |

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| mongodb 3.6 | passed |
| mongodb 4.0 | passed |
| mongodb 4.2 | passed |

File: pbm/restore/src.go [hunk 1]
```diff
@@ -12,6 +11,8 @@ import (
 	"github.com/aws/aws-sdk-go/aws/session"
 	"github.com/aws/aws-sdk-go/service/s3"
 	"github.com/golang/snappy"
+	"github.com/klauspost/compress/s2"
+	gzip "github.com/klauspost/pgzip"
```
Comment (akira-kurogane) [inline pbm/restore/src.go:15]:
```suggestion
	"gzip "github.com/klauspost/pgzip"
```

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| mongodb 3.6 | passed |
| mongodb 4.0 | passed |
| mongodb 4.2 | passed |

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| mongodb 3.6 | passed |
| mongodb 4.0 | passed |
| mongodb 4.2 | passed |

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| mongodb 3.6 | failed |
| mongodb 4.0 | passed |
| mongodb 4.2 | passed |

**Pull request merged**